### ISSUE TITLE
Make Rating Category Buttons

### ISSUE THREAD

**Issue by aarora4** [aarora4/VotingChromeExtension#2]
This task is responsible with making the button for users to filter the identified statements on a given page by their rating
<img width="116" alt="Screen Shot 2021-02-12 at 12 11 37 AM" src="https://user-images.githubusercontent.com/25010271/107732652-e1a5dc80-6cc6-11eb-81d3-ba19ac0a3b11.png">

This is the button that is being made. We will just duplicate it three times and change the color.

- [x] Make a button that has a circle, a label, and an outline. Lay out the button as shown in the picture
- [x] Make the button interact-able. **For now just make it print something to the console when you click it.**
- [x] Layout 3 of the buttons into a bar, as shown in the mockup below.

<img width="341" alt="Screen Shot 2021-02-12 at 12 08 28 AM" src="https://user-images.githubusercontent.com/25010271/107732698-fbdfba80-6cc6-11eb-9ecd-c100238cf088.png">

**Comment by aarora4** [comment]
Check here for information on how to get started making a button: https://developer.chrome.com/docs/extensions/mv2/user_interface/

**Comment by aarora4** [comment]
Test your UI code here: https://jsfiddle.net/
It'll be easier to just try different things here without loading the whole extension each time

**Comment by Imiri** [comment]
### Initial Work
<html>
  <head>
    <title>Buttons Popup</title>
  </head>
  <body>
      <button id='likelytrue'>Likely True</button>
      <button id='unknown'>Unknown</button>
      <button id='likelyfalse'>Likely False</button>
    <script src="popup.js"></script>
  </body>
</html>
//creates three popup buttons labeled Likely True, Unknown, and Likely False

**Comment by Imiri** [comment]
<html>
  <head>
    <title>Buttons Popup</title>
  </head>
  <body>
      <button id='likelytrue'>Likely True</button>
      <button id='unknown'>Unknown</button>
      <button id='likelyfalse'>Likely False</button>
    <script src="popup.js"></script>
  </body>
</html>

**Comment by Imiri** [comment]
<html>
<html>
  <head>
    <title>Buttons Popup</title>
  </head>
  <body>
      <button id='likelytrue'>Likely True 
      	<svg width="30" height="30">
   			<circle cx="15" cy="18" r="10" stroke="black" stroke-width="1" fill="green" />
	  	</svg> 
      </button>
      <button id='unknown'>Unknown 
      	<svg width="30" height="30">
   			<circle cx="15" cy="18" r="10" stroke="black" stroke-width="1" fill="gray" />
	  	</svg>
        </button>
      <button id='likelyfalse'>Likely False 
      	<svg width="30" height="30">
   			<circle cx="15" cy="18" r="10" stroke="black" stroke-width="1" fill="red" />
	  	</svg>
      </button>
    <script src="popup.js"></script>
  </body>
</html>
// creates three colored circles besides each button

**Comment by Imiri** [comment]
<html>
<html>
  <head>
    <title>Buttons Popup</title>
  </head>
  <body>
      <button id='likelytrue'>Likely True 
      	<svg width="30" height="30">
   			<circle cx="15" cy="18" r="10" stroke="black" stroke-width="1" fill="green" />
	  	</svg> 
      </button>
      <button id='unknown'>Unknown 
      	<svg width="30" height="30">
   			<circle cx="15" cy="18" r="10" stroke="black" stroke-width="1" fill="gray" />
	  	</svg>
        </button>
      <button id='likelyfalse'>Likely False 
      	<svg width="30" height="30">
   			<circle cx="15" cy="18" r="10" stroke="black" stroke-width="1" fill="red" />
	  	</svg>
      </button>
    <script src="popup.js"></script>
  </body>
</html>
//creates colored circles besides each button

**Comment by Imiri** [comment]
<button id='likelytrue'>Likely True 
      	<svg width="30" height="30">
   			<circle cx="15" cy="18" r="10" stroke="black" stroke-width="1" fill="green" />
	  	</svg> 
      </button>
      <button id='unknown'>Unknown 
      	<svg width="30" height="30">
   			<circle cx="15" cy="18" r="10" stroke="black" stroke-width="1" fill="gray" />
	  	</svg>
        </button>
      <button id='likelyfalse'>Likely False 
      	<svg width="30" height="30">
   			<circle cx="15" cy="18" r="10" stroke="black" stroke-width="1" fill="red" />
	  	</svg>
//creates three colored circles besides each button

**Comment by Imiri** [comment]
I don't think inline SVG is supported because the new code won't show up but basically I added the code for adding colored circles within each button.

**Comment by sabasid** [comment]
Ok here's what I got, is it similar to what you were doing Imiri?
*makes three buttons with colored circles and has them print something to console when clicked.
**They look a little different from the buttons on the mockup, but I figured cosmetic stuff is something we can keep working on. And if you have something different we can like change or combine or codes however we need to.

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

<button onclick="myFunction0()"><i class="fa fa-circle" style= "color:green"></i> Likely True </button>
<button onclick="myFunction1()"><i class="fa fa-circle" style= "color:gray"></i> Unsure</button>
<button onclick="myFunction2()"><i class="fa fa-circle" style= "color:red"></i> Likely False</button>

<p id="demo"></p>

<script>
function myFunction0() {
  document.getElementById("demo").innerHTML = "pls work";
}
function myFunction1() {
  document.getElementById("demo").innerHTML = "pls work again";
}
function myFunction2() {
  document.getElementById("demo").innerHTML = "pls work one last time";
}
</script>

**Comment by sabasid** [comment]
> Ok here's what I got, is it similar to what you were doing Imiri?
> *makes three buttons with colored circles and has them print something to console when clicked.
> **They look a little different from the buttons on the mockup, but I figured cosmetic stuff is something we can keep working on. We can like change or combine or codes as we need to 
> 
> Likely True
> Unsure
> Likely False
> 
> <script> function myFunction0() { document.getElementById("demo").innerHTML = "pls work"; } function myFunction1() { document.getElementById("demo").innerHTML = "pls work again"; } function myFunction2() { document.getElementById("demo").innerHTML = "pls work one last time"; } </script>

**Comment by Imiri** [comment]
I don't see the code for the buttons themselves so I can't see what they look like in jsfiddle; I don't know why, the same thing happened when I tried to comment my updated code in here. We could probably make our own branch of code and just merge them later.

**Comment by sabasid** [comment]
Yeah I'm not sure why it changes it's kinda weird, although I think u can see the original code if you press edit. But I'm fine with making our own and then seeing how we can merge it. 👍

**Comment by Imiri** [comment]
Its cool I just tried pressing edit and it worked like you said. Everything looks good to me and yeah we pretty much have the same stuff except yours actually prints something. You can replace my code with yours in the Rating Category Buttons section.

**Comment by sabasid** [comment]
Sure, I might play around with it a bit more tho, cause the way I got the circles to show up was a bit weird
